### PR TITLE
Created pull_request_template.md

### PR DESCRIPTION
**Pull request by aman2kumar**
Tried to solve Issue id = 42 (Create a pull request template)

### CODE REVIEW

File: docs/pull_request_template.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
+# Program Fixes:
+## Issue Id -
+Mention your issue ID here
```
Comment (Saransh-cpp) [inline docs/pull_request_template.md:3]:
```suggestion
## Issue
Fixes #(Mention the issue ID here)
```

File: docs/pull_request_template.md [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+
+<br>
+
+## Type of change
+You can write manually also
+
+- [ ] Bug fix (non-breaking change which fixes an issue)
+- [ ] New feature (non-breaking change which adds functionality)
+- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
+- [ ] This change requires a documentation update
```
Comment (Saransh-cpp) [inline docs/pull_request_template.md:19]:
```suggestion
- [ ] Documentation change
```

File: docs/pull_request_template.md [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+
+<br>
+
+## Screenshot
+Screenshot of the line where you found the bug
```
Comment (Saransh-cpp) [inline docs/pull_request_template.md:24]:
```suggestion
Screenshot of the fix if required.
```

File: docs/pull_request_template.md [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+
+## Checklist:
+
+- [ ] My code follows the style guidelines of this project
+- [ ] The app runs properly
+- [ ] I have performed a self-review of my own code
+- [ ] I have formatted and commented my code, particularly in hard-to-understand areas
+- [ ] I have made corresponding changes to the documentation
+- [ ] My changes generate no new warnings
+- [ ] I have added tests that prove my fix is effective or that my feature works
```
Comment (Saransh-cpp) [inline docs/pull_request_template.md:36]:
```suggestion
```

We don't have any tests yet :)

**Review by Saransh-cpp — CHANGES_REQUESTED** [review]
Thanks, @aman2kumar! Looks great:) Some minor comments below -

Also, please place this file in the `.github` folder, and rename it to `PULL_REQUEST_TEMPLATE.md`.

**Comment by Saransh-cpp** [comment]
Alos, could you please let me know the program that you are a part of?

**Comment by aman2kumar** [comment]
I am currently the part of CodePeak program. Thank you for your kind support and suggestions.

**Review by Saransh-cpp — APPROVED** [review]

**Comment by Saransh-cpp** [comment]
@all-contributors please add @aman2kumar for documentation

**Pull request merged**